### PR TITLE
Refactor embeddings builder in prep for SV addition

### PR DESCRIPTION
**Pull request by pradh**
This PR moves all data files to under `data/`, and allows multiple SV-name-description files.

* `data/curated_input/sheets_svs.csv`:  The current 1.3K spreadsheet which going forward will be edited only for "override descriptions"
* `data/autogen_input/*.csv`: Incremental additions of SVs will go here.
* `data/alternatives/*.csv`:  The palm and other alternatives can be in one or more files, and will go here.
* `data/preindex/sv_descriptions.csv`: The pre-index output for sanity check

In summary:  curated_input + autogen_input + alternatives => preindex => embeddings

### CODE REVIEW

**Comment by pradh** [comment]
Thanks Jehangir!
